src/Account.sol

```diff
@@ -79,53 +100,51 @@ contract Account is IAccount, Auth, OpsReady {
     /// @notice track conditional orders by id
     mapping(uint256 id => ConditionalOrder order) internal conditionalOrders;
 
+    /// @notice value used for reentrancy protection
+    uint256 internal locked = 1;
```

Comment:
**Nitpick:** I think this will initialise to 0 due to proxy setup, as the value 1 is not set in an initializer, but in the constructor on the implementation and then placed in storage. It shouldn't cause any problems because after the first call to `_dispatch` it will be set to 1. However it might be better to set it initially to 1 in an initializer function so that it doesn't have unexpected behaviour.

Reply:
agreed. this is sloppy and I actually fixed it already in v2.1.0 but will also fix it here